Classes/Repository/RepositoryIssuesViewController.swift

```diff
@@ -73,13 +75,32 @@ class RepositoryIssuesViewController: BaseListViewController<NSString>, BaseList
         }
     }
 
+    override func objects(for listAdapter: ListAdapter) -> [ListDiffable] {
+        let allObjects = super.objects(for: listAdapter)
```

Comment:
Accckkkkkkk I really really don't like this. Makes me regret making the base VC 😭 

I feel like we should probably bake filtering into the base VC and create a "search filterable" protocol on our models. I'll refactor all of this.

Reply:
Hehe, this was the best thing I could come up while working on it :)